components/clp-py-utils/clp_py_utils/clp_config.py

```
@@ -251,9 +260,30 @@ def transform_for_container(self):
         self.port = self.DEFAULT_PORT
 
 
+class SpiderDb(Database):
+
+    name: str = "spider-db"
+
+    @field_validator("type")
+    @classmethod
+    def validate_type(cls, value):
```

Ruff (0.14.4)

Missing return type annotation for classmethod `validate_type`

Add return type annotation: `None`

(ANN206)